src/Abstractions/RetryPolicy.cs

```diff
@@ -39,7 +47,8 @@ public class RetryPolicy
         TimeSpan firstRetryInterval,
         double backoffCoefficient = 1.0,
         TimeSpan? maxRetryInterval = null,
-        TimeSpan? retryTimeout = null)
+        TimeSpan? retryTimeout = null,
+        Func<TaskFailureDetails, bool>? handle = null)
```

Comment:
This is a binary breaking change. We can't include it in this ctor. I recommend using `get; init;` on the property, and have the init callback set a private field with `CreateHandler`

Reply:
Returned the constructor to its original parameters and created property `HandleTaskFailureDetails` that has no getter and a public `init`. Remove the static method `CreateHandler` and moved that code into the `init` callback. Note that the ctor code sets `Handle` to a delegate that will retry on all exceptions.